doc/modules/ensemble.rst

```diff
@@ -773,8 +833,8 @@ is too time consuming.
  * :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_oob.py`
  * :ref:`sphx_glr_auto_examples_ensemble_plot_ensemble_oob.py`
 
-Interpretation
---------------
+Interpretation with feature importance
```

Comment:
I think it could be good to add that it is entropy-based feature importance and not the permutation one

Reply:
done